src/ncoutputwriter.jl

```diff
@@ -0,0 +1,150 @@
+"""
+    WriteGeometry(model; filename="./geometry.nc", mode="c", slice_kw...)
+
+Writes a geometry.nc file that contains all the dimensions of the domain.
+
+Keyword arguments
+=================
+
+    - `filename::String`  : File name to be saved under
+    - `mode::String`      : Netcdf file is opened in either clobber ("c") or append ("a") mode (Default: "c" )
+"""
+function WriteGeometry(model; filename="./geometry.nc", mode="c", slice_kw...)
+    dimensions = Dict(
+        "xC" => collect(model.grid.xC),
+        "yC" => collect(model.grid.yC),
+        "zC" => collect(model.grid.zC),
+        "xF" => collect(model.grid.xF),
+        "yF" => collect(model.grid.yF),
+        "zF" => collect(model.grid.zF)
+    )
+
+    # Applies slices to the dimensions d
+    for (d, slice) in slice_kw
+        if String(d) in keys(dimensions)
+            dimensions[String(d)] = dimensions[String(d)][slice]
+        end
+    end
+
+    # Writes the sliced dimensions to the specified netcdf file
+    Dataset(filename, mode) do ds
+        for (dimname, dimarray) in dimensions
+            defDim(ds, dimname, length(dimarray)); sync(ds)
+            defVar(ds, dimname, dimarray, (dimname,))
+        end
+    end
```

Comment:
Does this function `return nothing`?

Reply:
Yes, it returns nothing. I'll add the `return` keyword.